api/src/main/java/org/pmiops/workbench/notebooks/NotebooksConfig.java

```diff
@@ -86,7 +86,7 @@ public ClusterApi serviceClusterApi(@Qualifier(NOTEBOOKS_SERVICE_CLIENT) ApiClie
   }
 
   private ApiClient buildApiClient(WorkbenchConfig workbenchConfig) {
-    return new ApiClient()
+    return new NotebooksApiClientTracer()
```

Comment:
We also have Mandrill and Jira API clients - may as well update those as well for consistency.

Reply:
Doing Mandrill, I don't see Jira here anymore. (at least, not under `api/src/main/java/org/pmiops/workbench`)